src/System.ComponentModel.Annotations/src/System/ComponentModel/DataAnnotations/AssociatedMetadataTypeTypeDescriptionProvider.cs

```diff
@@ -0,0 +1,32 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.ComponentModel.DataAnnotations
+{
+    public class AssociatedMetadataTypeTypeDescriptionProvider : TypeDescriptionProvider
```

Comment:
Can we copy over API docs as well?

Reply:
If you mean I can put xml docs on this class, then you're right. I can do that. But if I'm getting your mean incorrectly, then provide me more information. Thanks

Reply:
@ysmoradi that's right